ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -493,8 +524,42 @@ open class AvatarView: UIView {
         }
     }
 
+    private func updateImageViewWithPersonIcon(isSignedOut: Bool = false) {
+        let personImageSize = avatarSize.personImageSize
+        let containerSize = avatarSize.size
+        let imageName = isSignedOut ? "person_\(Int(personImageSize))_regular" : "person_\(Int(personImageSize))_filled"
+        if let image = UIImage.staticImageNamed(imageName) {
+            let renderer = UIGraphicsImageRenderer(size: containerSize)
+            let personImage = renderer.image { _ in
+                image.draw(at: CGPoint(x: floor((containerSize.width - personImageSize) / 2), y: floor((containerSize.height - personImageSize) / 2)))
+            }.withRenderingMode(.alwaysTemplate)
+            setupWithImage(personImage)
+            if isSignedOut {
+                imageView.backgroundColor = Colors.Avatar.signedOutImageBackground
+                imageView.tintColor = Colors.Avatar.signedOutImage
+            } else {
+                if let window = window {
+                    imageView.backgroundColor = Colors.primary(for: window)
+                }
+                imageView.tintColor = Colors.iconOnAccent
+            }
+            isPersonIconShown = true
+        }
+    }
+
+    private func shouldShowSignOutImage() -> Bool {
```

Comment:
Is this really the way that we want to derive the signed out state?

Reply:
do you think it should be more explicit?

Reply:
I think so but I can't think of an elegant way to do this.
Maybe create a setup method that's explicitly going to setup the AvatarView for the sign out case?

Reply:
It might make more sense to move this behavior outside of AvatarView.
It is possible to create an avatar view with any image.
I do think we can keep this logic in fluent UI to make it easier to create an AvatarView with the signed out image. You could subclass AvatarView to have a version that displays the sign out image. Subclassing is not the only way to do this. You could expose a static function that creates an AvatarView configured for the signed out state.

Reply:
you don't want to necessarily create a new avatar view because you are signed out. Header for example, you want to use the same view and swap out images depending on the authentication state. Avatar signed out state is part of the toolkit definition. So I do think it make sense to keep it in Fluent UI somewhere. I also, want to make sure we limit functionalities like presence but that's minor. out of all the ideas, I think setup() method with style (filled icon/outline icon) might be more abstract? what do you think?

Reply:
I still kind of feel like being signed in vs signed out isn't something the AvatarView should be concerned about. One potential use case of Avatar View is signed in/signed out representation for the active user of an app, but it's far from the only use case (see comments, coauthoring, presence bubbles, chat, etc). In iMessage, where an AvatarView equivalent exists, if I don't have a contact for someone I'm chatting with, that doesn't make them "signed out".

Reply:
I've been thinking more about this and I like the idea of using a more abstract concept than sign out.
I think that the existing logic for choosing the default icon works:
- If no primary or secondary text is provided, use the "signed out" image. 
- If no initials could be generated from the primary/secondary text, use the "default" image.

It's just a matter of finding a more generic term for the case where no user information is provided to AvatarView.
Just throwing ideas out there, we could go with: "FallbackImage" (or keep "DefaultImage") for the "default" case and "EmptyImage" for the "signed out" state.